apps/builder/app/builder/builder.tsx

```diff
@@ -319,12 +323,7 @@ export const Builder = ({
             <CanvasIframe
               ref={iframeRefCallback}
               src={canvasUrl}
-              pointerEvents={
-                dragAndDropState.isDragging &&
-                dragAndDropState.origin === "panel"
-                  ? "none"
-                  : "all"
-              }
+              pointerEvents={isCanvasPointerEventsEnabled ? "all" : "none"}
```

Comment:
Man says "all" is svg only. Better use "auto"
https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events